stories/topics/proc-azure-accessing-aap.adoc

```diff
@@ -9,8 +9,37 @@ Complete the form to provision {PlatformNameShort} infrastructure and resources
 . In a web browser, navigate to *Managed Applications* in the Azure console.
 . Select the instance of {AAPonAzureName} that you deployed.
 . Select *Parameters and Outputs* in the *Settings* section in the left navigation menu.
-. Copy the URL links for {ControllerName} and {HubName}, and then save them. The names for the links are _automationControllerUrl_ and _automationHubUrl_.
-. In a browser, navigate to the {ControllerName} URL, and then log in using the following credentials:
+**  The *Parameters and Outputs* page contains a link to the Welcome page. The Welcome page is available after deployment completes.
+From the Welcome page, you can access your {ControllerName} and {HubName} instance and view notifications about your deployment. You do not have to login to view the Welcome page.
+** The *Parameters and Outputs* page also displays direct links to {ControllerName} and {HubName}.
+. Save the URL links for the Welcome page, {ControllerName}, and {HubName}.
+The names for the links are _automationControllerUrl_, _automationHubUrl_, and _welcomePageUrl_.
+. Open the Welcome page.
+
+
+== Welcome page
+
+The Welcome page is a convenient landing page for private deployments of {AAPonAzureNameShort}.
+You can open the following views from the navigation pane:
+
+Overview:: Links to {ControllerName}, {HubName}, and Automation Analytics.
+You can disable the Welcome page by clicking **Manage view**.
++
+image::azure-welcome-page-overview.png[Welcome page overview]
+Automation Controller:: Displays links to {ControllerName} documentation.
++ 
+To open {ControllerName} from this view, click **Launch Automation Controller**.
+Automation Hub:: Displays links to {HubName} documentation. 
++
+To open {HubName} from this view, click **Launch Automation Hub**.
+Automation Analytics:: Displays links to {Automation Analytics} documentation
+Documentation:: Displays links to {AAPonAzureNameShort} documentation.
+
+To view notifications about {AAPonAzureNameShort}, click on the bell icon.
```

Comment:
Is there a bell icon?  As I'm icon hunting for controller i can look for that one.